docs/version0.33.md

```diff
@@ -0,0 +1,49 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# What's new in version 0.33.0
+
+The following new features and notable changes since version 0.30.0 are included in this release:
+
+- [New binaries and changes to supported environments](#binaries-and-supported-environments)
+- ![Start of content that applies to Java 11](cr/java11.png) [AIX requires the 16.1 XL C++ Runtime](#aix-requires-the-16-1-xl-c-runtime)
+
+## Features and changes
+
+### Binaries and supported environments
+
+OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
+
+### ![Start of content that applies to Java 11](cr/java11.png) AIX requires the XL C++ Runtime
+
+AIX requires the [16.1 XL C++ Runtime](https://www.ibm.com/support/pages/fix-list-xl-cc-runtime-aix#161X) to accommodate a security update to the HarfBuzz text shaping library.
+
```

Comment:
There's a mismatch between the link further up the topic and the title of this section, so the link doesn't work. But I think the words could use tweaking anyway to clarify that this is a change in level, to be more consistent with the text in the supported env topic, and to use a more official name for the runtime (which also clarifies that it's a third-party thing). Suggest something like:

> **Later level of XL C++ Runtime required on AIX**
> 
> AIX OpenJ9 builds now require version 16.1 of the IBM XL C++ Runtime for AIX (the previous required level was 13.1.3). This later level is required to accommodate a security update to the HarfBuzz text shaping library.

I also found this topic that mentions XL C++: https://eclipse-openj9.github.io/openj9-docs/configuring/#setting-resource-limits-aix-linux-and-macos. Looks like the Java 14 icon and text needs changing to 11.

Reply:
Ok, I saw the change in levels in the build env table but I guess that's different.